bin/collect_stats.py

```diff
@@ -95,26 +141,45 @@ def main(args=None):
 
         # condition_name, total_peptide_count
         total_peptide_count = sum(conditions_peptides["condition_peptide_count"])
-        print("Total peptides:", total_peptide_count, file=args.outfile, sep="\t", flush=True)
 
         # condition_name, unique_peptide_count
         unique_peptide_count = len(conditions_peptides)
-        print("Unique peptides:", unique_peptide_count, file=args.outfile, sep="\t", flush=True)
-        print(file=args.outfile)
-
-    # unique peptides across all conditions
-    all_conditions_unqiue_peptide_counts = len(protein_peptide_occs["peptide_id"].drop_duplicates())
-    print(file=args.outfile)
-    print(
-        "Unique peptides across all conditions:",
-        all_conditions_unqiue_peptide_counts,
-        file=args.outfile,
-        sep="\t",
-        flush=True,
-    )
 
-    print("Done!", flush=True)
+        # peptide_id, allele_id (with selected binders for allele in condition)
+        alleles_of_current_cond = conditions_alleles[conditions_alleles["condition_id"] == condition_id]["allele_id"].to_list()
+        predictions_condition_binders = predictions[predictions["allele_id"].isin(alleles_of_current_cond)]
```

Comment:
could you simply call it "binders" ? "predicitons_condition_binders" confused me a  bit :) (if you think this would be better, it's also fine!)